src/Stripe.net/Entities/Accounts/AccountCompany.cs

```diff
@@ -29,6 +29,15 @@ public class AccountCompany : StripeEntity<AccountCompany>
         [JsonProperty("directors_provided")]
         public bool DirectorsProvided { get; set; }
 
+        /// <summary>
+        ///  Whether the company's executives have been provided. Set this
+        ///  Boolean to `true` after creating all the company's executives with
```

Comment:
`true` should be `<c>true</c>` instead in dotnet and links should instead use
```
<a href="https://stripe.com/docs/subscriptions/webhooks">Using Webhooks with
        /// Subscriptions</a>.
```

Reply:
ack. fixing